tools/selinux/icinga2.fc

```diff
@@ -6,7 +6,8 @@
 
 /etc/icinga2/scripts(/.*)?	 --	gen_context(system_u:object_r:nagios_notification_plugin_exec_t,s0)
 
-/usr/sbin/icinga2		--	gen_context(system_u:object_r:icinga2_exec_t,s0)
+/usr/s?bin/icinga2		--	gen_context(system_u:object_r:icinga2_exec_t,s0)
```

Comment:
Did you test the latest version of this PR? Yes, I've suggested `/usr/s?bin/icinga2` and I expect it to work, but I haven't explicitly tested it.